menas/ui/service/EntityDialog.js

```diff
@@ -89,10 +89,18 @@ class DatasetDialog extends EntityDialog {
       this.oController.byId("newDatasetName"));
     let hasValidSchema = EntityValidationService.hasValidSchema(oDataset, "Dataset",
         this.oController.byId("schemaVersionSelect"));
-    let hasValidRawHDFSPath = this.oController.byId("newDatasetRawHDFSBrowser").validate();
-    let hasValidPublishHDFSPath = hasValidRawHDFSPath ? this.oController.byId("newDatasetPublishHDFSBrowser").validate() : false;
-
-    return hasValidName && hasValidSchema && hasValidPublishHDFSPath && hasValidRawHDFSPath;
+    let hasValidRawHDFSPath = EntityValidationService.hasValidHDFSPath(oDataset.hdfsPath,
+      "Dataset Raw HDFS path",
+      this.oController.byId("selectedRawHDFSPathLabel"));
+    let hasValidPublishHDFSPath = EntityValidationService.hasValidHDFSPath(oDataset.hdfsPublishPath,
+      "Dataset publish HDFS path",
+      this.oController.byId("selectedPublishHDFSPathLabel"));
+    let hasExistingRawHDFSPath = hasValidRawHDFSPath ? this.oController.byId("newDatasetRawHDFSBrowser").validate() : false;
+    let hasExistingPublishHDFSPath = hasValidRawHDFSPath && hasValidPublishHDFSPath ?
+      this.oController.byId("newDatasetPublishHDFSBrowser").validate() : false;
+
+    return hasValidName && hasValidSchema && hasValidPublishHDFSPath && hasValidRawHDFSPath
```

Comment:
```suggestion
    return hasValidName && hasValidSchema
```
These are covered by `hasExistingRawHDFSPath` and `hasExistingPublishHDFSPath`, so they can be removed.

Reply:
Please check. I also fixed it on line `268`